### ISSUE TITLE
Error which cmaking Zofi

### ISSUE THREAD

**Issue by 1Husin-Alhaj** [vporpo/zofi#2]
please, could you help me in fixing the below error 
![image](https://user-images.githubusercontent.com/17450624/88695839-b20ef880-d117-11ea-9685-8dfc1783a259.png)

**Comment by vporpo** [comment]
If I were to guess, your development environment is probably not 64-bits (x86_64). Zofi won't work on a 32-bit system.

If this is not the case, could you please provide more information about your development environment? For example:
- the name and version of the linux distribution you are using,
- the output of ldd --version

Thanks

**Comment by 1Husin-Alhaj** [comment]
yes, you are right, my system is 32-bit  ... thank you, I will test it on another env 
Thank for replay

**Comment by 1Husin-Alhaj** [comment]
After I installed another environment (Ubuntu 14 64-bit)
I got this errors:

![image](https://user-images.githubusercontent.com/17450624/88922075-da683580-d284-11ea-874e-02ff73d8b4e3.png)

Ubuntu release: 14.04.2  LTS
ldd (Ubuntu EGLIBC 2.19-0ubuntu6.5) 2.19
the g++ compiler version is 4.8.2

**Comment by vporpo** [comment]
Ah yes, I can reproduce the issue with gcc-4.8.2. It was indeed a missing header file. Thanks for reporting it, I just pushed the fix.
